lib/runnable.js

```diff
@@ -318,14 +318,15 @@ Runnable.prototype.run = function (fn) {
     self.clearTimeout();
     self.duration = new Date() - start;
     finished = true;
+    delete self.callback;
```

Comment:
what's this for?

Reply:
This is to ensure the `done()` function is only invoked once for a test